packages/node_modules/@webex/plugin-logger/src/logger.js

```diff
@@ -40,12 +40,20 @@ const authTokenKeyPattern = /[Aa]uthorization/;
 /**
  * Recursively strips "authorization" fields from the specified object
  * @param {Object} object
+ * @param {Array<mixed>} visited
```

Comment:
Could you mark this as optional since it has a default value?

```suggestion
 * @param {Array<mixed>} [visited]
```

Reply:
Updated